### ISSUE TITLE
Multiple values for code39 barcode

### ISSUE THREAD

**Issue by PhenixZeller** [juliansteenbakker/mobile_scanner#974]
I get multiple values when I scan a code39 type of barcode.

Flutter version: 3.19.0
Package version: 3.5.7

Example:
![Screenshot 2024-03-05 at 07 55 04](https://github.com/juliansteenbakker/mobile_scanner/assets/70652858/3b8beb27-72ba-4b55-a062-a7f9a8daa974)

Output onDetect:
```dart
onDetect: (capture) {
  for (final bc in capture.barcodes) {
    print('${bc.rawValue} || ${bc.displayValue} (format = ${bc.format}, type = ${bc.type})');
  }

// Returns
/*
20472E8
20472O
20472148
20472142 // Should be the only result
20472842
2047HE8
20472E
204724 
*/
```

Where do the values come from?

**Comment by navaronbracke** [comment]
Well, that depends on how MLKit detects the barcode. There isn't much we can do here.

**Comment by aungmyopaing890** [comment]
Hay @PhenixZeller,
Did you solved Multiple values problems,
i am also facing this error

**Comment by PhenixZeller** [comment]
> Hay @PhenixZeller, Did you solved Multiple values problems, i am also facing this error

No. I switched to another package which uses ZXing (Zebra Crossing) barcode scanning library instead of MLKit.

**Comment by juliansteenbakker** [comment]
Since this is an issue with the underlying MLKit framework, there is nothing much that we can do.
